src/module-1/styles/scss/base/_variables.scss

```diff
@@ -0,0 +1,9 @@
+//COLORS
+
```

Comment:
you may have separate css for colours and dimensions (widths, break-points, font sizes)